### PR TITLE
`HUGGING_FACE_HUB_TOKEN` -> `HF_TOKEN`

### PR DESCRIPTION
**Pull request by dakinggg**
HF deprecated the previous env var name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -264,7 +264,7 @@ Note: the `composer` command used above to train the model refers to the [Compos
 If you have a write-enabled [HuggingFace auth token](https://huggingface.co/docs/hub/security-tokens), you can optionally upload your model to the Hub! Just export your token like this:
 
 ```bash
-export HUGGING_FACE_HUB_TOKEN=your-auth-token
+export HF_TOKEN=your-auth-token
```
Comment (snarayan21) [inline README.md:267]: which version did they deprecate in? do we need a min version bump?

other than that, lgtm

**Review by snarayan21 — APPROVED** [review]
noice, maybe a min version bump doe

File: README.md (hunk 1 above)
Comment (dakinggg) [reply README.md:267]: they've supported both args for a while and still do, so no version bump needed.

**Review by mvpatel2000 — APPROVED** [review]
LGTM

**Pull request merged**